apps/prairielearn/src/lib/question.js

```diff
@@ -2,31 +2,32 @@
 
 const ERR = require('async-stacktrace');
 const _ = require('lodash');
-const async = require('async');
-const path = require('path');
-const jsonStringifySafe = require('json-stringify-safe');
-const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
-const ejs = require('ejs');
-const { differenceInMilliseconds, parseISO } = require('date-fns');
-const fs = require('fs');
-const unzipper = require('unzipper');
-const fg = require('fast-glob');
-const util = require('util');
-const { workspaceFastGlobDefaultOptions } = require('@prairielearn/workspace-utils');
-const { EncodedData } = require('@prairielearn/browser-utils');
-
-const { config, setLocalsFromConfig } = require('./config');
-const { generateSignedToken } = require('@prairielearn/signed-token');
-const externalGrader = require('./externalGrader');
-const manualGrading = require('./manualGrading');
-const ltiOutcomes = require('../lib/ltiOutcomes');
-const sqldb = require('@prairielearn/postgres');
-const questionServers = require('../question-servers');
-const workspaceHelper = require('./workspace');
-const issues = require('./issues');
-const error = require('@prairielearn/error');
-const { createServerJob } = require('./server-jobs');
-
+import * as async from 'async';
+import * as path from 'path';
+import * as jsonStringifySafe from 'json-stringify-safe';
```

Comment:
This (and possibly others here) will be runtime errors: this code tries to call `jsonStringifySafe`, but a namespace import will *never* be directly callable. I'd also recommend installing TS types for this package.

Reply:
Resolved (and types installed) in 29ef820.

I also checked the other imports and it seems the others are used appropriately.